common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -0,0 +1,33 @@
+param(
+    $Framework = 'netcoreapp2.1' # 'netcoreapp2.1' or 'net461'
```

Comment:
Is this necessary? For our cases the latest version for either framework should be the same.